packages/app/ios/Orbit/IngestEventEmitter.h

```diff
@@ -0,0 +1,30 @@
+//
+//  EventEmitter.h
+//  Orbit
+//
+//  Created by Ozzie Kirkby on 2022-05-07.
+//
+
+#ifndef EventEmitter_h
+#define EventEmitter_h
+
+#import <React/RCTBridgeModule.h>
+#import <React/RCTEventEmitter.h>
+
+typedef enum {
+    kSuccess,
+    kFailure,
+    kWaiting
+} IntentState;
+
+@interface IngestEventEmitter : RCTEventEmitter <RCTBridgeModule>
+
+@property(atomic,assign) bool hasListeners;
```

Comment:
There isn't a consistent style for spacing in this syntax, but I'd suggest making 24 look like 22/23.